src/routes/balances/balances.service.ts

```diff
@@ -81,6 +81,7 @@ export class BalancesService {
   }
 
   async getSupportedFiatCodes(): Promise<string[]> {
+    // TODO: take into consideration BalancesRepository available fiat codes.
```

Comment:
I left a TODO here as I think this can be done in a separate PR to avoid creating too much noise.